lib/push-jenkins-update.js

```diff
@@ -0,0 +1,80 @@
+'use strict'
+
+const githubClient = require('./github-client')
+
+const jenkinsToGhStatusMap = {
+  'SUCCESS': {
+    state: 'success',
+    message: 'all tests passed'
+  },
+  'FAILURE': {
+    state: 'failure',
+    message: 'build failure'
+  },
+  'ABORTED': {
+    state: 'error',
+    message: 'build aborted'
+  },
+  'UNSTABLE': {
+    state: 'error',
+    message: 'build unstable'
+  }
+}
+
+const rxDisplayName = /node\-test\-commit\-(\w+)/
```

Comment:
I don't think the linter has this name, is there that much use in regexing this?